src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumModule.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.bridge.sdk;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.Deserializers;
+import com.fasterxml.jackson.databind.deser.Deserializers.Base;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+@SuppressWarnings("serial")
+class LowercaseEnumModule extends SimpleModule {
```

Comment:
ScheduleStrategyTest.canRountripSimplePlan is an example of a test that serializes and deserializes an enum as part of the object, and then compares the two objects to verify they are equal. There are many such tests, but I can write one that just tests the enum serializer/deserializer alone.